whylog/tests/tests_front/tests_utils.py

```diff
@@ -0,0 +1,19 @@
+from whylog.front.utils import FrontInput, LocallyAccessibleLogOpener
+
+
+def tests_unit_creation_sample_front_input():
+    offset = 123
+    line_content = 'I like trains'
+    path = '/sample/path/to/file'
+    resource_location = LocallyAccessibleLogOpener(path)
+    front_input = FrontInput(offset, line_content, resource_location)
+
+    assert front_input.offset == offset
+    assert front_input.line_content == line_content
+    assert front_input.resource_location is resource_location
```

Comment:
Maybe remove this test? It tests only constructor. Bugs in constructor will quickly come out in any serious test which uses FrontInput.